src/ReaderScreen/components/autoScrollComponent.jsx

```diff
@@ -17,11 +17,13 @@ function AutoScrollComponent({ shabadID }) {
 
   const handleSpeed = (value) => {
     dispatch(setAutoScrollSpeed(value, shabadID));
-    if (value === 0) togglePaused(true);
+    if (value === 0) {
+      togglePaused(true);
+    }
   };
   const handlePause = () => {};
   const handlePlay = () => {};
-  const handleScroll = () => {};
+  // const handleScroll = () => {};
```

Comment:
if we don't need this code anymore please remove it

Reply:
fixed